Daemon/Daemon/Monitor.m

```diff
@@ -506,6 +506,18 @@ -(void)processEvent:(File*)file
         goto bail;
     }
 
+    //passive mode?
+    // no rule, just allow (but log)
+    if(YES == [preferences.preferences[PREF_PASSIVE_MODE] boolValue])
```

Comment:
Check for passive mode can be made before `Event` instance is created. Losing a little bit of logging, i.e. `-[Event description]` provides representation of `Item`, which prints `[event.plugin itemName:event];` and `[event.plugin itemObject:event];`. 

For your consideration.